### PR TITLE
Add ServiceAccountName, remove outer for loop, update default restart count

### PR DESCRIPTION
**Pull request by tremaineeto**
We need ```serviceAccountName: kubernetes-deployment-crawler``` in the cronjob.yaml or else k8s will use the ```default``` ServiceAccount. Credit @gbraxton with that.

This PR also removes the outer for loop in main.go.

Lastly, this PR also updates the default restart threshold to 144.

### CODE REVIEW

**Comment by timjonesdev** [comment]
> We need `serviceAccountName: kubernetes-deployment-crawler` in the cronjob.yaml or else k8s will use the `default` ServiceAccount. Credit @gbraxton with that.
> 
How does this service account get created? Will this fail in someone's cluster if they don't have this service account already? Should we provide some setup instructions for this?

File: main.go [hunk 1]
```diff
@@ -70,46 +70,44 @@ func main() {
```
Comment (timjonesdev) [inline main.go:70]: what are the changes in this file? Seems like this PR should only be the yaml file?

**Comment by tremaineeto** [comment]
Good question @timjonesdev. There is another .yaml file in this folder called serviceaccount.yaml. It would be created in the same ```kubectl apply -f install/``` command.

**Review by gbraxton — APPROVED** [review]

File: main.go (hunk 1 above)
Comment (tremaineeto) [reply main.go:70]: I'm slightly confused by this too, but I compared the "new" main.go code here with master, and it seemed the same. I'm not sure why it shows these changes. Any thoughts?

Comment (timjonesdev) [reply main.go:70]: did you remove the outer for-loop?

Comment (tremaineeto) [reply main.go:70]: I did, yes.

**Review by timjonesdev — APPROVED** [review]

**Pull request merged**